go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/go-git/go-git/v5 v5.4.1
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/mock v1.5.0
-	github.com/golang/protobuf v1.4.3
+	github.com/golang/protobuf v1.5.2
```

Comment:
Was this necessary for the ginkgo bump? Is there a chance this will make things hairier when rolling out to other repos?

Reply:
I will explore. All of these deps were behind and I intentionally updated them to match what we have in our projects (GE, GME). I will make sure that we can pull in these changes successfully and if there are challenges will reconsider

Reply:
If it's bringing the dep up to date with our other repos that's great
I'd be nervous if it was pushing a dep ahead of other repos